lib/Backend/Inline.cpp

```diff
@@ -1928,7 +1950,7 @@ Inline::InlineBuiltInFunction(IR::Instr *callInstr, const FunctionJITTimeInfo *
     IR::Instr *inlineBuiltInEndInstr = nullptr;
     if (inlineCallOpCode ==  Js::OpCode::InlineFunctionApply)
     {
-       inlineBuiltInEndInstr = InlineApply(callInstr, inlineeData, inlinerData, symCallerThis, pIsInlined, profileId, recursiveInlineDepth);
+       inlineBuiltInEndInstr = InlineApply(callInstr, inlineeData, inlinerData, symCallerThis, pIsInlined, profileId, recursiveInlineDepth, inlineCallArgCount - 1);
```

Comment:
Because I want to pass in the actual number of arguments passed to the apply call. inlineeCallArgCount takes into account the this pointer as well for a call to apply.

---
In reply to: [85817969](https://github.com/Microsoft/ChakraCore/pull/1838#discussion_r85817969) [](ancestors = 85817969)